app/renderer/actions/Session.js

```diff
@@ -724,6 +729,52 @@ export function setSavedServerParams () {
   };
 }
 
+export function setStateFromAppiumFile (newFilepath = null) {
+  return async (dispatch) => {
+    // no "fs" means we're not in an Electron renderer so do nothing
+    if (!fs) {
+      return;
+    }
+    try {
+      let filePath = newFilepath;
+      if (!newFilepath) {
+        const lastArg = process.argv[process.argv.length - 1];
+        if (!lastArg.startsWith('filename=')) {
+          return;
+        }
+        filePath = lastArg.split('=')[1];
+      }
+      if (sessionStorage.getItem(FILE_PATH_STORAGE_KEY) === filePath) {
+        // file was opened already, do nothing
+        return;
+      }
+      const appiumJson = JSON.parse(await util.promisify(fs.readFile)(filePath, 'utf8'));
```

Comment:
What should here do when `filePath` was invalid or not existed?
Just I clone this change and started with `npm run dev`, then I got the below error.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/5511591/166191077-29a614b6-6ac7-47eb-864a-8d2500c4baa3.png">

The `filePath` was `./`. Is it when a user open an appium inspector without selecting any appiumsession files before?

Reply:
I'll change it so that it doesn't launch a file from the command line when it's dev. The problem is that dev is launched as `<electron> --inspect ./` and it's picking up `./`.